lib/autokey/scripting/window.py

```diff
@@ -258,25 +262,26 @@ def center_window(self, win_width=None, win_height=None, title=":ACTIVE:", monit
         # 1: +DVI-D-0 1440/408x900/255+1920+0  DVI-D-0
         # Regex gets  ____ and ____    ____ _
         # this translates to monitor x and y size, and x and y offset for each monitor
-        rcode, output = self._run_xrandr(["--listactivemonitors"])
+        returncode, output = self._run_xrandr(["--listactivemonitors"])
         regex = r" (\d{3,4}).*?x(\d{3,4})\/.*?\+(\d{1,4})\+(\d{1,4})"
         matches = re.findall(regex, output, re.MULTILINE)
-        x_offset = int(matches[monitor][2])
-        y_offset = int(matches[monitor][3])
-        width = int(matches[monitor][0])
-        height = int(matches[monitor][1])
+
+        width, height = [int(i) for i in matches[monitor][0:2]]
+        x_offset, y_offset = [int(i) for i in matches[monitor][2:4]]
```

Comment:
This is way less clear than the previous version. Don't hide details under complicated python syntax just to save a few lines, IMO. Especially in a project like this, where maintainers may not know python well.